lerna.json

```diff
@@ -8,6 +8,12 @@
       "registry": "https://registry.npmjs.org"
     }
   },
-  "ignoreChanges": ["**/__test__/**", "**/__demo__/**", "**/__snapshots__/**", ".vscode/**"],
+  "ignoreChanges": [
```

Comment:
Did anyone suggest you to update `ignoreChanges`? I don't think it's a good idea as [lerna docs](https://github.com/lerna/lerna/tree/main/libs/commands/version#--ignore-changes) say that there are some cases that a package will always be published, regardless of this option.

Reply:
So that we will let Solar theme update when Elemental theme is changed, isn't it?

Reply:
You're right. I will look for proper way to ignore the change in package.

Reply:
The goal is to clearly communicate that the package is deprecated. Releasing new versions might contradict with the deprecation but I don't think it's a big deal.

In short, I believe there is no need to stop publishing new versions.